libfuse/mounter.go

```
@@ -20,20 +20,51 @@ type mounter struct {
 	c       *fuse.Conn
 }
 
+const darwinInstallerPath = "/Applications/Keybase.app/Contents/Resources/" +
+	"KeybaseInstaller.app/Contents/MacOS/Keybase"
+
+func (m *mounter) reinstallMountDirForDarwin() {
+	args := []string{
+		"--app-path=/Applications/Keybase.app",
+		"--run-mode=prod",
+		"--timeout=60",
+	}
+	exec.Command(darwinInstallerPath,
```

Ignoring err values here -- can we at least print them?